src/script/common/c_content.h

```diff
@@ -107,6 +108,11 @@ ItemStack          read_item                 (lua_State *L, int index, IItemDefM
 
 struct TileAnimationParams read_animation_definition(lua_State *L, int index);
 
+PointabilityType   read_pointability         (lua_State *L, int index);
+PointingAbilities  read_pointabilities(lua_State *L, int index);
+void push_pointability(lua_State *L, const PointabilityType& pointable);
+void push_pointabilities(lua_State *L, const PointingAbilities &pointabilities);
```

Comment:
Names are kinda confusing. Maybe `read_pointability_type` instead of `read_pointability`?

Reply:
fixed